src/python/pants/option/options_test.py

```diff
@@ -1142,6 +1142,64 @@ def test_passthru_args_not_interpreted():
     ).string
 
 
+def test_alias() -> None:
```

Comment:
There is ample testing of the aliasing itself on the rust side, but this is a good test that it all fits together correctly in the bootstrapping.